app/controllers/admin/filter_tags_controller.rb

```diff
@@ -5,14 +5,26 @@ class GlobalFilter::AdminFilterTagsController < Admin::AdminController
   def index
     render_serialized(
     { filter_tags: GlobalFilter::FilterTag.all.order(:name) },
-      GlobalFilter::AdminFilterTagIndexSerializer,
+      GlobalFilter::FilterTagIndexSerializer,
       root: false
     )
   end
 
   def set_category_ids_for_tag
-    params.require([:tag])
+    params.require(:tag)
     filter_tag = GlobalFilter::FilterTag.find_by(name: params[:tag])
     filter_tag.update!(category_ids: params[:category_ids]&.join("|") || "")
   end
+
+  def set_alternate_name_for_tag
+    params.require(:tag)
+    filter_tag = GlobalFilter::FilterTag.find_by(name: params[:tag])
+    filter_tag.update!(alternate_name: params[:alternate_name] || nil)
+  end
+
+  def set_alternate_composer_only_for_tag
```

Comment:
```suggestion
  def set_alternate_composer_only_name_for_tag
```

Very minor, only suggesting because I am finding it hard to read `set_alternate_composer_only_for_tag`. 

Another option might be to merge these values and permit the  `:alternative_composer_only)` param.